src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -1130,6 +1130,7 @@ Thread.State threadState() {
                 }
                 // runnable, mounted
                 return Thread.State.RUNNABLE;
+	    case BLOCKING:
```

Comment:
Please fix the indentation here (uses spaces only).

Reply:
Fixed.